src/selectors/columnWidths.js

```diff
@@ -105,54 +111,59 @@ function columnWidths(
  *   columnProps: !Array.<columnDefinition>
  * }}
  */
-function flexWidths(columnGroupProps, columnProps, viewportWidth) {
-  let newColumnProps = columnProps;
-  let remainingFlexGrow = getTotalFlexGrow(columnProps);
-
-  // if any column is a flex column, we'll need to calculate the widths for every column
-  if (remainingFlexGrow !== 0) {
-    const columnsWidth = getTotalWidth(columnProps);
-    let remainingFlexWidth = Math.max(viewportWidth - columnsWidth, 0);
-
-    // calculate and set width for each column
-    newColumnProps = map(columnProps, (column) => {
-      const { flexGrow } = column;
-
-      // if no flexGrow is specified, column defaults to original width
-      if (!flexGrow) {
-        return column;
+function flexWidths(columnGroupElements, columnElements, viewportWidth) {
```

Comment:
The main change here is that the `flexWidths` also calculates the offsets of the column based on the new flex width.

This was previously done by `groupElements()` below.